server/venueless/live/modules/auth.py

```diff
@@ -44,6 +41,9 @@ async def login(self):
         await self.consumer.channel_layer.group_add(
             f"user.{self.consumer.user['id']}", self.consumer.channel_name
         )
+        await self.consumer.channel_layer.group_add(
+            f"world.{self.world}", self.consumer.channel_name
```

Comment:
I wonder if we can find a way to document the groups we use, because this is starting to take up my working memory. Either a list somewhere or a data abstraction? Or just depend on the fact that the list won't grow that much and that we can remember what we are using?

Reply:
We should at least list it, yes!